great_expectations/expectations/core/expect_column_values_to_be_between.py

```diff
@@ -108,7 +108,7 @@ class ExpectColumnValuesToBeBetween(ColumnMapExpectation):
         "profiler_config",
     )
 
-    min_estimato_parameter_builder_config: ParameterBuilderConfig = (
+    min_estimator_parameter_builder_config: ParameterBuilderConfig = (
```

Comment:
Minor typo - LMK if this should be moved to another PR.

Reply:
🙇